_posts/2021-01-16-sr2021-first-league-sessions.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: news
+title: SR2021 First League Session a Great Success!
+---
+
+After the first 36 matches of the SR2021 Competition, team HRS3 from
+Hills Road Sixth Form College head up the league leaderboard with a
+staggering 82 game points. They're closely followed by the other 3 Hills Road
+teams as well as one of our rookie teams We Robots.
+
+The league matches provide a proving ground for all the teams' hard work over
+the past few months in preparing their robot code. There is also a prize for the
+rookie team which scores the best in the league, which both the We Robots team
+and Harris Westminster Sixth Form Team 1 are currently well placed for.
+
+At the end of the league we announced to teams the details of our second game
```

Comment:
Should probably link out to the rules page somewhere here.

Reply:
Note: the link to the rules isn't going to change, so we should probably add this in now so we're not waiting for the rules changes themselves.

Reply:
https://github.com/srobo/website/pull/253/commits/a2a0c33449bb1b77b014eb224c283650d923a3c6